pkg/minishift/openshift/oc.go

```diff
@@ -17,8 +17,13 @@ limitations under the License.
 package openshift
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
 	instanceState "github.com/minishift/minishift/pkg/minishift/config"
 	"github.com/minishift/minishift/pkg/util"
+	routeApi "github.com/openshift/origin/pkg/route/api"
```

Comment:
Do we really need to import this. We are pretty much free of dependencies to the OpenShift and probably could remove it.